src/ray/protobuf/node_manager.proto

```diff
@@ -310,10 +310,18 @@ message GetObjectsInfoReply {
   repeated CoreWorkerStats core_workers_stats = 1;
 }
 
+message GetResourceLoadRequest {}
+
+message GetResourceLoadReply {
+  ResourcesData resources = 1;
```

Comment:
Why don't we separate resource loads fields from `ResourcesData`?

Reply:
Because it's deep coupled with autoscaler and not a trivial change. I don't think it fits into this PR.